core/src/main/scala/org/genivi/sota/core/campaigns/CampaignLauncher.scala

```diff
@@ -66,4 +87,5 @@ object CampaignLauncher {
       _ <- db.run(Campaigns.setAsLaunch(id))
     } yield updateRefs
   }
+  //scalastyle:on parameter.number
```

Comment:
Yes, unfortunately, due to all the implicits, we are over 8 parameters. Leaving the warning means a huge block of code is marked yellow, which hurts readability in Intellij, and also encourages potentially ignoring other, valid, warnings.